packages/design-system/src/components/InputField/InputField.js

```diff
@@ -13,6 +13,8 @@ const InputField = React.forwardRef((props, ref) => {
     className,
     id,
     fieldClassName,
+    maxLength,
+    charsLeft,
```

Comment:
I asked Justyna about the [design](https://www.figma.com/file/KCYyNivnhN4wSDOM0Qjas4/KnowledgeBase-onboarding?node-id=845%3A6022). It looks like on the right it should be possible to add also something different (ie. link) so those props looks not right. 

How about a different approach - let's start exporting hooks from the library! `useUsedCharactersCounter` hook seems awesome imho. You could use it in your app and pass the prepared text to the component with new props `labelRightNode`. How it sounds?